<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_teethmatch_tool</class>
 <widget class="QDialog" name="Dialog_teethmatch_tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>配齿工具</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/advancedsettings.ico</normaloff>icons/advancedsettings.ico</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>571</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>参数</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>501</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>最小齿数</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>速比</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_speed_ratio"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>螺旋角</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_beta_angle">
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>速比允许误差</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="comboBox_speed_ratio_deviation">
       <item>
        <property name="text">
         <string>±5%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>±10%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>±15%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>±20%</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="spinBox_gearteeth_min">
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="radioButton_actual_center_distance">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>中心距</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_normal_module">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>法向模数</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_d_or_mn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>计算方案</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>540</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>601</width>
     <height>311</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>中心轮齿数</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>行星轮齿数</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>法向模数</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>螺旋角</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>中心距</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>速比</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_teethmatch_tool</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
